source/Ansible-RabbitMQ-Tests/RabbitMQClientTest.class.st

```diff
@@ -399,15 +434,131 @@ RabbitMQClientTest >> testPublishingMessages [
 ]
 
 { #category : 'tests' }
-RabbitMQClientTest >> testPublishingOneMessage [
+RabbitMQClientTest >> testPublishingToDefaultExchange [
 
-	self resumeWorkerDuring: [ publisher publishOnly: 'Hello' onQueueNamed: self queueName ].
+	self resumeWorkerDuring: [ publisher publish: 'Hello' to: self queueName ].
 
 	self
 		withTheOnlyOneIn: reversedTexts
 		do: [ :reversedText | self assert: reversedText equals: 'olleH' ]
 ]
 
+{ #category : 'tests' }
+RabbitMQClientTest >> testPublishingToDirectExchangeWithTwoQueuesBindedWithDifferentRoutingKey [
+
+  |  firstWorkerMessages secondWorkerMessages |
+
+  firstWorkerMessages := OrderedCollection new.
+  secondWorkerMessages := OrderedCollection new.
+
+  self
+    runWorkerNamed: 'Reverser'
+    consumingFrom: 'reverser-queue'
+    bindedTo: 'amq.direct'
```

Comment:
The name 'amq.direct' is well a well-known name, maybe it's a good idea to do an extract method.